solr/CHANGES.txt

```diff
@@ -140,6 +140,9 @@ Improvements
 
 * SOLR-17096: solr.xml now supports declaring clusterSingleton plugins (Paul McArthur, David Smiley)
 
+* SOLR-17119: When registering a ConfigurablePlugin through the `/cluster/plugin` API using the
+  V2Request client, config validation exceptions are now propagated to the callers (Yohann Callea)
```

Comment:
Suggest rewording "the V2Request client" to "SolrJ with V2Request".   (people know SolrJ is a client and this word sets better context to users who have no clue what V2Request even is).

Curious; is"V2Request" actually pertinent to the bug?  Would someone using plain HTTP see the problem?

Reply:
It would indeed also affect users querying the API in plain HTTP without SolrJ.
I agree that the proposed description seems to imply that the issue only exist when calling the API through SolrJ, I updated it to be less specific.

I also moved the change to 9.6.0 as requested in the previous comment.

Thanks again for your guidance 🙂